### DISCUSSION TITLE
Having difficulty adding .delta files as synteny tracks through the command line

### DISCUSSION THREAD

**Discussion by carolyncaron** [GMOD/jbrowse-components#3628] [Q&A] [answered]
Hello,

I have 10+ .delta files generated using nucmer that I want to add as synteny tracks (and many more files are coming). I updated my JBrowse2 version to v2.4.2 since I noticed that v2.4.1 mentions adding functionality for adding synteny tracks through the CLI, but I am getting errors.

Here is what I've tried based on commands in the docs:
```
jbrowse add-track 1GRN-1DRT.delta --type SyntenyTrack --assemblyNames Lcu.1GRN,Ler.1DRT --load symlink
›   Error: Unexpected arguments: --type, SyntenyTrack
›   See more help with --help
```
```
jbrowse add-track 1GRN-1DRT.delta --assemblyNames Lcu.1GRN,Ler.1DRT --load symlink
 ›   Error: Track type is not recognized
```

I tried adding the same delta file through the user interface and it works. It also works by updating the config.json directly - but I'd really like to avoid doing it this way for all of them!

Is there a parameter in the command that I'm missing? Any suggestions for how to get this to work?

**Comment by cmdcolin** [comment, marked as answer]
One thing that may be of note is that you would want your CLI tools to be up to date. The version of 'jbrowse-web' (what you see in the browser) can be different from the version of the 'CLI tool' (what the `jbrowse` command is)

to update the CLI tools (the `jbrowse` command), can use

```
npm install -g @jbrowse/cli
```

Then run 

```
jbrowse --version
```

To confirm the CLI tool version

**Reply by cmdcolin** [reply]
I can run similar command with the latest version (add a delta file) and I think it should work. I am hoping to write some more tutorials on loading different types of synteny data soon, as we have only really documented minimap2/PAF file loading somewhat, but the CLI tool should support delta after you get it updated. Example command I see locally

```
$ jbrowse add-track volvox_inv_indels.delta --out test --assemblyNames volvox,volvox_inv --load symlink

Added track with name "volvox_inv_indels" and trackId "volvox_inv_indels" to test/config.json


```

**Reply by cmdcolin** [reply]
and just as another note, if you see e.g. the data is empty after loading, can try switching the order of --assemblyNames  you pass in

**Comment by carolyncaron** [comment]
Thanks so much for your quick reply! My CLI tool was indeed out of date so updating it allowed me to run the second command I tried successfully. 👍

**Comment by rbuels** [comment]
I'm wondering @cmdcolin if it would be worth having the CLI tool check whether it is out of date and give a warning if so? `yarn` does this I think.

**Reply by cmdcolin** [reply]
could make an issue for it. the CLI is a pretty slow moving target and may not need frequent updates, but occasionally updating can help e.g. in things like this
